Dev7/TriangleMain.java

```diff
@@ -0,0 +1,30 @@
+import java.util.InputMismatchException;
+
+public class TriangleMain {
+  public static void main(String[] args) {
+    InputSides input = new InputSides();
+    double[] array = input.inputSides();
+    Triangle triangle = new Triangle(array);
```

Comment:
you don't need it as every triangle type calls Triangle constructor (and corresponding check for existent)